client/src/components/my-mentor-card/index.tsx

```diff
@@ -37,6 +37,7 @@ const useStyles = makeStyles(() => ({
 export default function MyMentorCard(props: {
   accessToken: string;
   mentor: Mentor;
+  buildAction: (mentorId: string) => void;
```

Comment:
seems like mentor is already known and fixed by the caller (since they're passing a mentor).

So seems like `buildAction` should not require (or even accept) a `mentorId` param?